.github/ISSUE_TEMPLATE/request-to-be-elector-for-toc-election.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Request to be elector for TOC election
+about: Requests for community members to be added as electors for a TOC election cycle.
+title: Request to be an elector for TOC election
+labels: election
+assignees: ''
```

Comment:
Based on a decision in #74, we could create a team for toc members and add it as assignee (or does that work differently than PR reviewers and assignees have to be dedicated people?).